augur/config.py

```diff
@@ -129,125 +157,123 @@
                     "model": "pull_request_analysis",
                     "repo_group_id": 0
                 },
-
-	        {
-	    	"delay": 10000,
-		"given":[
-		    "git_url"
-		],
-		"model" : "discourse_analysis",
-		"repo_group_id" : 0
-
-	        },
-	        {
-                "delay": 10000,
-                "given": [
-                    "git_url"
-                ],
-                "model": "clustering",
-                "repo_group_id": 0
+    	        {
+	    	        "delay": 10000,
+            		"given":[
+            		    "git_url"
+            		],
+            		"model" : "discourse_analysis",
+            		"repo_group_id" : 0
+	           },
+	            {
+                    "delay": 10000,
+                    "given": [
+                        "git_url"
+                    ],
+                    "model": "clustering",
+                    "repo_group_id": 0
```

Comment:
This indentation is incorrect. The brackets appear too far too the left, and the code appears to be too far to the right.